CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Changelog: [mojaloop/api-snippets](https://github.com/mojaloop/api-snippets)
+## [14.1.0](https://github.com/mojaloop/api-snippets/compare/v14.0.0...v14.1.0) (2022-07-15)
```

Comment:
@mdebarros 
That was a fresh "git pull && git branch" setup.
And it did break several times due to git hooks triggering.

The first commit had to contain lots of information about the changes. I did "git commit -a", populated the description, saved it, and found out that the commit wasn't done due to an incorrect message format and the entire commit message was gone. Next time I decided not to write it again and used just a short description.
When I tried to do "git commit" after that, it populated the description with some autogenerated "chore... something". I removed it and closed the message editor without saving it, but on exit from the message editor, the commit was created anyway.
I decided to amend the commit message, but this time used a short description instead in case it won't be saved again.
But it did change my git name to "mojaloopci" in commits after that and bumped the version number in package.json.

Tbh, this workflow (when some scripts **automatically** can remove your work or change/generate files) frustrate me a lot.
It just forces me to avoid creating extra commits for not to break some stuff.
And always keeping in mind to add --no-verify to every commit command is not an option either.

Reply:
> Tbh, this workflow (when some scripts automatically can remove your work or change/generate files) frustrate me a lot.

Agreed. This should be addressed. 

As discussed, please add a TODO section in the README where you can add some of your suggested improvements.

I think the most important thing here (as part of your suggestion): All generated files should be done during build time (i.e. when publishing or building docker images, etc).